src/main/java/mario/Player.java

```diff
@@ -30,23 +27,21 @@
     private final List<Key> keys = new ArrayList<>();
     private final GameDashboard gameDashboard;
 
-    private int walkingSpeed = 5;
-    private int jumpingSpeed = 8;
-    private int numberOfkeysPickedUp = 0;
+    private final int walkingSpeed = 5;
```

Comment:
Had ik bewust niet final gemaakt omdat we een power up gingen maken waardoor Mario sneller zou bewegen.